open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/factory/GraphFactory.java

```diff
@@ -63,44 +62,17 @@ public JanusGraph openGraph(String graphDB,ConnectionProperties connectionProper
         }
     }
 
-    /**
-     * Initialize Janus Graph.
-     *
-     * @param janusGraph  - Janus Graph instance
-     * @param graphType - type of the graph to be initiated
-     * @return JanusGraph instance
-     */
-    private JanusGraph initialize(JanusGraph janusGraph,String graphType){
-        final String methodName = "initialize";
```

Comment:
Method was redundant since all it did was calling a single other method. I've moved the log messages to the other methods.